api/src/clojure/org/akvo/flow_api/datastore/form_instance.clj

```diff
@@ -103,9 +104,14 @@
   [_ response-str]
   response-str)
 
+(def ^DateFormat sdf (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z"))
```

Comment:
It seems that declaring the var as `:^dynamic`, and using `(binding [date-format (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z")] ...)` or just declare the new SimpleDateFormat object inside a `let` ?